### PR TITLE
Migrate to ESLint

### PR DESCRIPTION
**Pull request by zckrs**
* bc2bbb8 Add ESLint dep
* 9fda9c5 Fix ESLint error produce by `npm test`
* e1243b9 Remove .jshintrc

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,8 +32,15 @@
     "q": "^1.4.1"
   },
   "devDependencies": {
+    "eslint": "^1.10.3",
     "mocha": "^2.2.5"
   },
+  "eslintConfig": {
```
Comment (sindresorhus) [inline package.json]: Since this is non-standard, I would put it at the bottom.

Comment (zckrs) [inline package.json]: Done

**Comment by sindresorhus** [comment]
Looks good. Thanks :)

File: index.js [hunk 2]
```diff
@@ -23,10 +23,10 @@ var gconfig = {
 if (keyFilename) {
   gconfig.keyFilename = keyFilename;
 }
-var bucket = gcloud(gconfig).storage().bucket(bucket);
+bucket = gcloud(gconfig).storage().bucket(bucket);
```
Comment (eddiemonge) [inline index.js]: why did you remove the `var`?

Comment (sindresorhus) [inline index.js]: @eddiemonge It's already defined on line 13.

Comment (eddiemonge) [inline index.js]: Oh I forgot this is the old code.
